manual/03-lists/Transforming.md

```diff
@@ -2,3 +2,46 @@ List to list transformations
 ============================
 
 TODO: `map`, `pluck`, `filter`, `aperture`, `uniq`, `zip`, `partition`
+
+Many times we loop over lists for...
+--
+
+**Transforming**
+```js
+const libs = ["Ramda", "Lodash", "Pramda", "Underscore"]
+const html = []
+for(var i = 0; i < libs.length; i++){
+  html[i] = "<li>"+libs[i]+"</li>"
+}
+console.log(html)
+
+```
+Above code is too verbose and we are prone to making mistakes. Ramda provides a `map` function to achieve
+  
+```js
+const libs = ["Ramda", "Lodash", "Pramda", "Underscore"]
+R.map((lib)=>"<li>"+lib+"</li>", libs)
+```
+
+Much more concise. We could refactor this code further by writing a reusable wrapTag function.
```

Comment:
Concision is not a goal in itself; clarity should be. This is clearer because we don't need to know which element we are transforming (e.g. `libs[i]`)